recipe/meta.yaml

```diff
@@ -13,8 +13,12 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 2
   skip: true  # [win]
+  run_exports:
+    # Looks good but no new info after 2.43 so keeping it x.x for safety.
+    # https://abi-laboratory.pro/?view=timeline&l=librsvg
+    - {{ pin_subpackage('librsvg', max_pin='x') }}
```

Comment:
The code and the comment seem inconsistent here?

Reply:
Yep. I updated one but not the other. Sorry.